manifests/piped/templates/deployment.yaml

```diff
@@ -59,6 +61,12 @@ spec:
         - name: piped-config
           configMap:
             name: {{ include "piped.configMapName" . }}
+        - name: ssh-config
```

Comment:
Do we need this volume?

Reply:
Btw, we should change the default value of `ssh config path`.
https://github.com/pipe-cd/pipe/blob/master/pkg/config/piped.go#L165

By switching to a non-root user, piped will be failed with the current default value.